backend/scripts/sync_prod_to_sqlite.py

```
@@ -0,0 +1,288 @@
+def latest_dump():
+    dumps = sorted(BACKUP_DIR.glob('tally_prod_*.sql'), key=lambda p: p.stat().st_mtime)
+    if not dumps:
+        sys.exit('No dump found in backups/. Run without --reuse-dump.')
+    return dumps[-1]
+
+
+def pg_container_exists():
+    return bool(capture(['docker', 'ps', '-aq', '-f', f'name=^{CONTAINER}$']))
+
+
+def start_postgres(fresh=False):
```

Ruff (0.16.0)

[warning] Boolean default positional argument in function definition

(FBT002)